### DISCUSSION TITLE
For books: When the metadata or cover file is not available, use the first page as a fallback strategy

### DISCUSSION THREAD

**Discussion by brokedarius** [jellyfin/jellyfin#12206] [General]
Hey folks I going back and forth between jellyfin and kavita and noticed that jellyfin currently has no fall back strategy of that sort and no scrapers for books either so I think that could be a nice compromise since often times the first page will be the cover anyways and in the rare occurences where its not the user could always resort to the top strategy that would be the cover file or metadata.

This is useful especially for stuff that has been packaged in unusual ways or where not such file was provided.

**Comment by mkalpakis** [comment]
bump

**Comment by GeiserX** [comment]
Some time ago I tried installing some plugin for it, but it was not working. 
I'd really like this to be baked in the Jellyfin base project as an option in the system config. I'm spending literally hours just on putting the cover from the first page of the book (in whatever format it is, ePub or PDF) when this could be easily done automatically.
